### ISSUE TITLE
[New Feature]: ECMWF data retrieval

### ISSUE THREAD

**Issue by hhlee445** [nasa/opera-sds-pcm#643]
### Checked for duplicates

Yes - I've already checked

### Alternatives considered

No - I haven't considered

### Describe the feature request

OPERA SDS needs to retrieve ECMWF weather data into S3 bucket for DISP-S1 processing and deliver to ASF DAAC.

Sources of ECMWF weather data
 - Historical processing mode: Using ECMWF API  
 - Forward processing mode: From ASF DAAC which is delivered by NISAR project

Format of ECMWF weather data
- Download and delivery: GRIB2
- Processing: NetCDF

ECMWF data retrieval instruction is available upon request. 

ADT team would want to get every model time (i.e. every 6 hours) for an entire month.

**Comment by philipjyoon** [comment]
@hhlee445 Is OPERA SDS delivering ECMWF weather data separately to ASF in addition to the DISP-S1 products?

**Comment by hhlee445** [comment]
Yes. We need to deliver  ECMWF weather data as separate products.

**Comment by hhlee445** [comment]
There is an instance on opera-ecmwf host and has been downloading ECMWF data from 2015 to 2024.
